cmds/tcz/tcz.go

```diff
@@ -202,19 +210,21 @@ func installPackage(tczName string, deps map[string]bool) error {
 }
 
 func setupPackages(tczName string, deps map[string]bool) error {
-	l.Printf("setupPackages: @ %v deps %v\n", tczName, deps)
+	debug("setupPackages: @ %v deps %v\n", tczName, deps)
 	for v := range deps {
-		cmdName := strings.Split(v, ".")[0]
+		cmdName := strings.Split(v, filepath.Ext(v))[0]
 		packagePath := path.Join("/tmp/tcloop", cmdName)
 		if err := os.MkdirAll(packagePath, 0700); err != nil {
-			l.Fatal(err)
+			l.Printf("overwriting %s", packagePath)
```

Comment:
I think we . need a better message. This is *really* confusing. is it good? bad? I can't tell.